ionic/components/modal/test/basic/index.ts

```diff
@@ -1,9 +1,11 @@
 import {App, Page, Config, Platform} from '../../../../../ionic';
-import {Modal, ActionSheet, NavController, NavParams, Transition, TransitionOptions, ViewController} from '../../../../../ionic';;
+import {Modal, ActionSheet, NavController, NavParams, Transition, TransitionOptions, ViewController} from '../../../../../ionic';
 
+import {Http} from 'angular2/http';
 
 @Page({
-  templateUrl: 'main.html'
+  templateUrl: 'main.html',
+  providers : [Http]
```

Comment:
Good point.  I was trying to think of a service to inject to test and make sure the dynamically added component could still get stuff from the provider.  I tried making my own service/provider but Typescript wasn't happy about it.  Can you think an injectable service to test with?